components/tidb_query_executors/src/runner.rs

```diff
@@ -459,9 +460,6 @@ impl<SS: 'static> BatchExecutorsRunner<SS> {
                 sel_resp.set_warnings(warnings.warnings.into());
                 sel_resp.set_warning_count(warnings.warning_cnt as i64);
 
-                // In case of this function is called multiple times.
-                self.exec_stats.clear();
```

Comment:
Why remove this?

Reply:
Because I need to collect the exec_stats after the `handle_request`, if exec_stats clear here, the exec_stats will become empty